swaybar/ipc.c

```diff
@@ -153,8 +153,10 @@ static bool ipc_parse_config(
 		return false;
 	}
 	json_object *markup, *mode, *hidden_state, *position, *status_command;
-	json_object *font, *bar_height, *wrap_scroll, *workspace_buttons, *strip_workspace_numbers;
-	json_object *binding_mode_indicator, *verbose, *colors, *sep_symbol, *outputs;
+	json_object *font, *bar_height, *wrap_scroll, *workspace_buttons, 
+		*strip_workspace_numbers, *strip_workspace_name;
```

Comment:
Follow what was done previously and start a new line of declarations `json_object *...` instead of overhanging it.

Also remember to fix trailing spaces.